pkger/models.go

```diff
@@ -773,12 +774,13 @@ func (c chart) properties() influxdb.ViewProperties {
 			Note:              c.Note,
 			ShowNoteWhenEmpty: c.NoteOnEmpty,
 		}
-	case chartKindScatter:
+	case chartKindHeatMap:
 		ia := c.Axes.influxAxes()
-		return influxdb.ScatterViewProperties{
```

Comment:
I just alphabetized things here. didn't introduce anything new besides the reading in the Markdown props.